rust/operator-binary/src/crd/affinity.rs

```diff
@@ -5,7 +5,7 @@ use stackable_operator::{
     k8s_openapi::api::core::v1::{PodAffinity, PodAntiAffinity},
 };
 
-use crate::{AirflowRole, APP_NAME};
+use crate::crd::{AirflowRole, APP_NAME};
```

Comment:
Is `AirflowRole` supposed to be versioned?

Reply:
We _can_ version it if we want. I focused on the CRD and the cluster config for all my PRs.

Reply:
Cool, no problem